### PR TITLE
Modification to add a simple chroot environment for people who don't have access to a USB Flash Drive

### PR DESCRIPTION
**Pull request by dk865**
This is a very small script I worked on, a modification of the `build.sh` script, that instead of builds a disk image, creates a chroot environment that can be accessed via another shell script easily (that it creates.)

### CODE REVIEW

**Comment by claration** [comment]
Whats the point of this?

**Comment by dk865** [comment]
Sorry, it's for people (on Linux) who don't have a Flash Drive available to install palen1x. For some reason on (a fresh install) of Ubuntu 24.04 LTS, the Linux Binaries and Debian Package both don't work, however my chroot solution does for me. I made it because of an issue that was brought up with someone having the same problem, but it was closed, so I couldn't give it to them. It's okay if you don't add it, but I thought it would be a nice small addition.

**Comment by dk865** [comment]
I have noticed very minor issues, such as you can't exit using power off or reboot, but you can hold `ctrl` and press `c` to exit, and the Shell option in the menu doesn't work, but I have plans to fix these issues. You may want to wait a bit before adding though if you do so that I can fix it first. Please tell me if you do not plan to add it though so that I can abandon it, or maybe just create a new tool altogether.

**Comment by dk865** [comment]
Just a heads up, if you're going to test it on Linux, if you can't exit, and need to terminate it, make sure to unmount everything, or `sudo` won't work, and will give you an error such as `unable to allocate PTY` afterwards. If you can't figure out how to unmount (I think palera1n developers would know, it was even in the build script) you can run the last lines of the script manually, or simply reboot.

**Comment by claration** [comment]
While we appreciate the efforts put into this, this really is just a half-baked solution to a problem people are facing on popular distributions. This "solution" also requires someone to be in an environment to be able to setup the chroot, which not everyone has since the build script requires dependences from APT, which is usually found on Debian based distributions.

Secondly, including this "solution" will require us to maintain support for it and give commands for something I find unnecessarily complex.

Alternatively, when it comes to trying to fix support for Ubuntu or any other distribution facing this issue, it's recommended you find the root cause of it and contribute fixing it.

In conclusion this is more suited for a fork, sorry.

**Comment by dk865** [comment]
Okay, thanks for your time. Have a good one!
